lib/OpenQA/Script/CloneJobSUSE.pm

```diff
@@ -28,6 +30,9 @@ sub verify_incident_repos ($url_handler, $incident_repos) {
     my @incident_urls;
     my $ua = $url_handler->{ua};
     foreach my $incident (split(/,/, $incident_repos)) {
+        die
```

Comment:
I don't think it is good idea , maybe just print warning instead ? It reminds me story around https://github.com/os-autoinst/openQA/pull/6468 which showed us that it is not always good thing to decide on behalf of caller if it is make sense to continue . I still some cases where no matter that incident repos are not expanded run still make sense . For example some VRs where I simply skip use of this variables

Reply:
I wouldn't mind the `die`. If we continue here we would probably just run into a hard failure later so failing fast is probably the best thing to do here.

Reply:
I've just double-checked and code in `CloneJobSUSE.pm` is only executed if `--check-repos` is present.